.travis.yml

```diff
@@ -31,12 +31,12 @@ env:
         # Set defaults to avoid repeating in most cases
         - PYTHON_VERSION=3.6
         - NUMPY_VERSION=stable
-        - MAIN_CMD='python setup.py'
         - CONDA_DEPENDENCIES='Cython'
         - CONDA_ALL_DEPENDENCIES='astropy Cython'
         - SETUP_XVFB=True
         - EVENT_TYPE='push pull_request'
-        - SETUP_CMD='test'
+        - INSTALL_CMD='python setup.py build_ext --inplace'
+        - TEST_CMD='pytest'
```

Comment:
I wouldn't recommend calling `pytest` directly. Use `setup.py test` as before. The latter invokes some custom test runner magic.